kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtWifiChannel.java

```diff
@@ -25,7 +25,8 @@
     netWifiChannel10,
     netWifiChannel11,
     netWifiChannel12,
-    netWifiChannel13;
+    netWifiChannel13,
```

Comment:
I think you forgot to change the header in this file, it should be "Copyright (c) 2011, 2020 Eurotech and/or its affiliates"